### ISSUE TITLE
Looking for a animator/producer to create a deep dive into calculus video.

### ISSUE THREAD

**Issue by bpt246** [leios/SoME_Topics#149]
Hello! I'm Bennett and I'm looking for a animator/producer to help me get the message of calculus across to the youth. The main point I want to take a deep dive into is derivates and how they can be used and possibly abused to how you would like, giving derivates more of a realistic view over the more simplistic view can help children really connect with the teacher and the subject.

Thank you.

**Comment by Ghostly726** [comment]
Hey There!
I'd like to help you create some animations. Do you have anything specific in mind? Specific topics or examples etc
Thanks in advance

**Comment by Ghostly726** [comment]
oh so you like want me to mess around with the topic and have fun with it, while also establishing some relativity with it and real life topics?

**Comment by bpt246** [comment]
I would personally love everyone working on it(right now it is just me, you and maybe my friend) to have free will(but contained in a calculus setup). Several topics still relating to that main idea would be perfect. From free to use your own ideas in the final copy. 

More of a mathematical portion-
Do you believe that the derivative thing I was showing you proves that repeating derivatives are fundamentally about factorials(when using it in conjunction with the power rule)?

Thank you! :D

**Comment by Ghostly726** [comment]
I just tried this with other powers and it works! But I could not find any proofs online. It is interesting regardless

I'd like to start collaborating with you

**Comment by bpt246** [comment]
Yes! I was working with other powers and adding some multiplication and stuff like that into the equation. Do you know somewhere we could message privately?

**Comment by Ghostly726** [comment]
my email
abdulrahmankhaledal32021@gmail.com

**Comment by elancha98** [comment]
Hey,

Maybe a bit late to post this here, but the idea you guys are working on about deriving multilple times is closely related to the idea of fractional derivatives. More precisely, the relation you guys are posting is where the idea of a fractional derivative comes. It has to do with the factorial and Cauchy's formula for repeated integration.

I leave a few links here so you guys can look them up.

[Fractional derivatives](https://en.wikipedia.org/wiki/Fractional_calculus)
[Cauchy's formula for repeated integration](https://en.wikipedia.org/wiki/Cauchy_formula_for_repeated_integration)

Whether you use this info or not, I find this topic interesting and I think you guys will do an amazing job. Keep up the work and I'm looking forward to seen the final result :)

**Comment by Ghostly726** [comment]
ohh very interesting!

**Comment by bpt246** [comment]
Thank you so much for contributing
